var/spack/repos/builtin/packages/pacifica-cli/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PacificaCli(PythonPackage):
+    """Python CLI for Pacifica Core Services"""
+
+    homepage = "https://github.com/pacifica/pacifica-cli/"
+
+    version('0.5.2', sha256='fee5fa8ac38ffec2e9199bff23afbbae697c4f7554f13e340104f8b20a62843f', url="https://pypi.io/packages/source/p/pacifica-cli/pacifica-cli-0.5.2.tar.gz")
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-wheel', type='build')
+    depends_on('py-importlib-metadata', type='run')
+    depends_on('py-jsonschema', type='run')
+    depends_on('py-requests', type='run')
```

Comment:
Same here, this might not be needed

Reply:
py-wheel and jsonschema left in, as they are needed. requests moved to the pacifica-downloader files